src/backend/core/cart/views.py

```diff
@@ -133,24 +136,39 @@ class CartUpdateAddressBaseStorefrontView(APIView, ABC):
     Do not use this view directly, use inherited classes instead (and implement `_set_address` method)
     """
 
+    address_serializers = (
```

Comment:
> It should be called address_serializer, right ? Because the s at the end is confusing me, and suggests that there may be multiple ones, but from the rest of the code, it doesn't look like that

yes, sorry, my fault, it should be in a singular form


> Also, why are there parentheses around None ?

It's work of a formatter 😄